cmd/test-containerd/test_containerd.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
move / rename "cmd/test-containerd/test_containerd.go" to "test/containerd/containerd.go"

Reply:
Done.